integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -24,47 +24,65 @@ describe('Native ETH value integration tests', () => {
     other = Wallet.createRandom().connect(wallet.provider)
   })
 
-  it('should allow an L2 EOA to send to a new account and back again', async () => {
-    const getBalances = async (): Promise<BigNumber[]> => {
-      return [
-        await wallet.provider.getBalance(wallet.address),
-        await wallet.provider.getBalance(other.address),
-      ]
-    }
-
-    const checkBalances = async (
-      expectedBalances: BigNumber[]
-    ): Promise<void> => {
-      const realBalances = await getBalances()
-      expect(realBalances[0]).to.deep.eq(expectedBalances[0])
-      expect(realBalances[1]).to.deep.eq(expectedBalances[1])
-    }
+  describe('Basic behavior', () => {
+    it('should allow an L2 EOA to send to a new account and back again', async () => {
+      const getBalances = async (): Promise<BigNumber[]> => {
+        return [
+          await wallet.provider.getBalance(wallet.address),
+          await wallet.provider.getBalance(other.address),
+        ]
+      }
+
+      const checkBalances = async (
+        expectedBalances: BigNumber[]
+      ): Promise<void> => {
+        const realBalances = await getBalances()
+        expect(realBalances[0]).to.deep.eq(expectedBalances[0])
+        expect(realBalances[1]).to.deep.eq(expectedBalances[1])
+      }
+
+      const value = 10
+      await fundUser(env.watcher, env.l1Bridge, value, wallet.address)
+
+      const initialBalances = await getBalances()
+
+      const there = await wallet.sendTransaction({
+        to: other.address,
+        value,
+        gasPrice: 0,
+      })
+      await there.wait()
 
-    const value = 10
-    await fundUser(env.watcher, env.l1Bridge, value, wallet.address)
+      await checkBalances([
+        initialBalances[0].sub(value),
+        initialBalances[1].add(value),
+      ])
 
-    const initialBalances = await getBalances()
+      const backAgain = await other.sendTransaction({
+        to: wallet.address,
+        value,
+        gasPrice: 0,
+      })
+      await backAgain.wait()
 
-    const there = await wallet.sendTransaction({
-      to: other.address,
-      value,
-      gasPrice: 0,
+      await checkBalances(initialBalances)
     })
-    await there.wait()
 
-    await checkBalances([
-      initialBalances[0].sub(value),
-      initialBalances[1].add(value),
-    ])
+    it('should allow ETH to be burned by sending to zero address', async () => {
```

Comment:
Sorry for the messed up diff.  The actual test starts here, the rest was just moving these two into a new `describe`.